src/shared/common/forms/type-ahead.jsx

```diff
@@ -83,6 +96,7 @@ class TypeAhead extends Component {
                     error={error}
                     onChange={this.handleChange.bind(this)}
                     loadOptions={this.getOptions.bind(this)}
+                    defaultOptions={defaultOptions}
```

Comment:
is this okay to be `null`? (probably)

Reply:
Yes, in fact only one widget has it defined. Both undefined and false would mean the same thing in this instance.